moped-editor/src/queries/project.js

```diff
@@ -401,6 +408,32 @@ export const ADD_PROJECT_MILESTONE = gql`
   }
 `;
 
+export const PROJECT_FOLLOW = gql`
+  mutation FollowProject(
+    $objects: [moped_user_followed_projects_insert_input!]!
+  ) {
+    insert_moped_user_followed_projects(objects: $objects) {
+      returning {
+        id
+        project_id
+        user_id
+      }
+    }
+  }
+`;
+
+export const PROJECT_UNFOLLOW = gql`
+  mutation UnfollowProject($project_id: Int!, $user_id: Int!) {
+    delete_moped_user_followed_projects(where: { project_id: { _eq: $project_id }, user_id: { _eq: $user_id } }) {
+      returning {
+        id
+        project_id
+        user_id
+      }
```

Comment:
I don't think it makes sense to ask for `graphql-engine` to return data about a record that was deleted. Can this be removed?

Reply:
@frankhereford I see your point but I don't know if it's possible not to return any data. At least every way I have tried results in a syntax error. This [article](https://stackoverflow.com/questions/44737043/is-it-possible-to-not-return-any-data-when-using-a-graphql-mutation) makes it seem a bit complicated. Can you please advise?

Reply:
Actually I was able to rework it like this. Is this closer to what you have in mind?

```
  mutation UnfollowProject($project_id: Int!, $user_id: Int!) {
    delete_moped_user_followed_projects(where: { project_id: { _eq: $project_id }, user_id: { _eq: $user_id } }) {
      affected_rows
    }
  }
```

Reply:
I noted it I think in another comment, but to your question, this is perfect IMHO.